initCobraToolbox.m

```diff
@@ -636,19 +658,37 @@ function checkGit()
             end
         end
     end
-
+    
     if ENV_VARS.printLevel
         fprintf(' > Checking if remote can be reached ... ')
     end
-
+    
     % check if the remote repository can be reached
     [status_curl, result_curl] = system('curl -s -k --head https://github.com/opencobra/cobratoolbox');
-
+    
     % check if the URL exists
     if status_curl == 0 && ~isempty(strfind(result_curl, ' 200'))
         if ENV_VARS.printLevel
             fprintf(' Done.\n');
         end
+    elseif (status_curl == 127 || status_curl == 48) && (strcmp(computer('arch'), 'glnxa64') || strcmp(computer('arch'), 'maci64'))
```

Comment:
using `isunix` should do the trick here no? (instead of `(strcmp(computer('arch'), 'glnxa64') || strcmp(computer('arch'), 'maci64'))`

Reply:
Could you add a comment here explaining the error codes `127` and `48`?